lib/src/fnc/util/math/bottom.rs

```diff
@@ -1,11 +1,30 @@
 use crate::sql::number::Number;
+use std::collections::BinaryHeap;
 
 pub trait Bottom {
-	fn bottom(self, _c: i64) -> Number;
+	/// Find the lowest `k` records from the collection in arbetrary order  
+	/// O(n*k*log(k)) time complex 
+	fn bottom(self, _c: i64) -> Vec<Number>;
 }
 
 impl Bottom for Vec<Number> {
-	fn bottom(self, _c: i64) -> Number {
-		todo!()
+	fn bottom(self, _c: i64) -> Vec<Number> {
```

Comment:
Variable names that began with `_` (which indicates they weren't used) that are now actually used should be promoted to not use `_`